app/uk/gov/hmrc/exports/models/Choice.scala

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.exports.models
+import play.api.libs.json._
+
+sealed abstract class Choice(val value: String)
```

Comment:
This would be a lot less code if we used Scala Enumerations e.g. 

https://github.com/hmrc/customs-declare-exports/blob/master/app/uk/gov/hmrc/exports/models/declaration/DeclarationStatus.scala

You can give them String Values  too e.g.

```
object Choice extends Enumeration {
  type Choice = Value
  implicit val format: Format[Choice.Value] = EnumJson.format(Choice)
  val Standard = Value("STD")
  val Supplementary = Value("SMP")
}
```

Reply:
As discussed we are going with a Sealed Trait. The added effort of writes and reads seem unnecessary when compared to an Enum, but the wider community tend to prefer sealed traits. Let's see where this takes us.